src/app/fdctl/config/default.toml

```diff
@@ -1021,9 +1021,24 @@ dynamic_port_range = "8900-9000"
         # second.  When benchmarking, this can be the limiting
         # bottleneck of the system, so this option is provided to raise
         # the limit.  If set to true, the limit will be lifted to
-        # 288,000,000 for a little over 1 million transactions a second.
+        # 288,000,000 for a little over 1 million cheap transactions a
```

Comment:
Sorry my fault. Isn't this actually 480,000 TPS? (6*80,000) Can we update the number to be over 1M ?

Reply:
I updated it to 13x, which should be over 1M